src/store/composesSlice.js

```diff
@@ -53,6 +53,7 @@ export const selectComposeById = (state, composeId) => {
       created_at: compose.created_at,
       id: compose.id,
       imageName: compose.image_name || compose.id,
+      imageDescription: compose.image_description || compose.id,
```

Comment:
I don't think we want to use the compose id in case there is a no description, maybe it would be better to use an empty string or `none`.